eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/exporter/zipkinExporter.java

```diff
@@ -17,8 +17,9 @@
 package org.apache.eventmesh.runtime.exporter;
 
 import io.opentelemetry.exporter.zipkin.ZipkinSpanExporter;
+import org.apache.eventmesh.common.config.CommonConfiguration;
 
-public class zipkinExporter {
+public class zipkinExporter implements EventMeshExporter {
```

Comment:
`zipkinExporter` isn't right, change to `ZipkinExporter`

Reply:
ok